src/index.es6

```diff
@@ -230,3 +230,7 @@ export default class ApiaryLibGenerator extends Base {
     git.setEmail(this.data.email);
   }
 }
+
+
+// This has no equivalent in ES2015 and it is needed for Yeoman to work
+module.exports = ApiaryLibGenerator;
```

Comment:
I consider `module.exports` to be a Node thing, so I don't see any problem using it in libraries. If you web pack or browserify, you won't use native import/export anyway.

:+1: